ddl/reorg.go

```diff
@@ -553,12 +569,12 @@ func getTableRange(ctx *JobContext, d *ddlCtx, tbl table.PhysicalTable, snapshot
 		endHandleKey = tablecodec.EncodeRecordKey(tbl.RecordPrefix(), maxHandle)
 	}
 	if isEmptyTable || endHandleKey.Cmp(startHandleKey) < 0 {
-		logutil.BgLogger().Info("[ddl] get noop table range",
+		logutil.BgLogger().Info("[ddl] get table range, endHandle < startHandle",
 			zap.String("table", fmt.Sprintf("%v", tbl.Meta())),
 			zap.Int64("table/partition ID", tbl.GetPhysicalID()),
-			zap.String("start key", hex.EncodeToString(startHandleKey)),
-			zap.String("end key", hex.EncodeToString(endHandleKey)),
-			zap.Bool("is empty table", isEmptyTable))
+			zap.Bool("isEmptyTable", isEmptyTable),
+			zap.String("endHandle", hex.EncodeToString(endHandleKey)),
+			zap.String("startHandle", hex.EncodeToString(startHandleKey)))
```

Comment:
This seems to be an unexpected change?

Reply:
Right, it was changed during debugging, let me remove that change.